generatedocs/data/config/scheduler/description.md

```diff
@@ -1,3 +1,17 @@
 ## Scheduler section
 
 This section of the JSON file contains scheduler settings for the users in the load scenario.
+
+### Using "reconnectsettings"
+
+If `reconnectsettings.reconnect` is enabled the tool will attempt the following:
```

Comment:
Suggestion:
"If `reconnectsettings.reconnect` is enabled, the following is attempted:"